src/vm/ecalllist.h

```diff
@@ -652,13 +652,13 @@ FCFuncStart(gMathFuncs)
     FCIntrinsic("Cosh", COMDouble::Cosh, CORINFO_INTRINSIC_Cosh)
     FCIntrinsic("Exp", COMDouble::Exp, CORINFO_INTRINSIC_Exp)
     FCIntrinsic("Floor", COMDouble::Floor, CORINFO_INTRINSIC_Floor)
+    FCFuncElement("fmod", COMDouble::FMod)
     FCFuncElement("Log", COMDouble::Log)
     FCIntrinsic("Log10", COMDouble::Log10, CORINFO_INTRINSIC_Log10)
+    FCFuncElement("modf", COMDouble::ModF)
     FCIntrinsic("Pow", COMDouble::Pow, CORINFO_INTRINSIC_Pow)
-    FCIntrinsic("Round", COMDouble::Round, CORINFO_INTRINSIC_Round)
```

Comment:
This means that Round will be no longer treated as intrinsic by the JIT.

Reply:
What is the mechanism used to treat a non FCALL method as intrinsic?

Reply:
Mark the method with `[Intrinsic]` attribute and recognize it by name in the JIT.

#13815 introduced this mechanism.

Reply:
Alright, thanks!

I've already added the `[Intrinsic]` attribute, so I should just need to recognize it by name.

Reply:
@jkotas, I ended up adding two new named intrinsics (one for `Math.Round` and one for `MathF.Round`).

I then modified the Compiler::impIntrinsic in importer to map the new named intrinsic to the old CORINFO intrinsic.

This looks to be working end to end right now, since the named intrinsic checks only happen in morph and importer, but only before the `GenTree*` has been created.